buildSrc/src/main/kotlin/GradleUtil.kt

```diff
@@ -26,26 +26,26 @@ val String.sysProp: String?
    get() = System.getProperty(this)?.takeIf { it.isNotBlank() }
 
 open class LinkJDK: DefaultTask() {
-   /** Not used, but supply it anyway to trigger the task if java version changes. */
+   /** Not used directly, but useful to automatically trigger the task if the java version changes. */
    @Input lateinit var jdkVersion: JavaVersion
-   /** Directory/link that will contain the JDK. Should be inside the project. */
-   @OutputDirectory lateinit var linkDir: File
+   /** Location of the link to the JDK. */
+   @Input @Internal lateinit var linkLocation: File
 
    @TaskAction
    fun linkJdk() {
-      linkDir.delete() // delete invalid symbolic link
-      println("Linking JDK to project relative directory...")
+      linkLocation.delete() // delete invalid symbolic link
       val jdkPath = "java.home".sysProp?.let { Paths.get(it) } ?: failIO { "Unable to find JDK" }
+      logger.info("Creating link at $linkLocation to $jdkPath...")
       try {
-         Files.createSymbolicLink(linkDir.toPath(), jdkPath)
+         Files.createSymbolicLink(linkLocation.toPath(), jdkPath)
       } catch (e: Exception) {
-         println("Couldn't create a symbolic link from $linkDir to $jdkPath: $e")
+         logger.warn("Couldn't create a symbolic link at $linkLocation to $jdkPath: $e")
```

Comment:
You can run grade with --info and --debug to see the respective messages, this prevents polluting the log with trivial messages. It is Gradles logger mechanism, and you should be aware of the advantages of such a thing already ;)

Reply:
I wasnt aware.

Running with --info is a bit of a hassle (especially from IDE). Also the prints are only present if the task actually runs so I feel like that accomplishes the job fairly well, but alright... let's use that.

I dont think this 'consistency' will prove to have added value though - because you do not usually use trace/debug and suddenly you lose the info about what the task is doing - this is even more palpable if you are first timer cloning the project.

Reply:
If you "are first timer cloning the project" then you don't want random messages showing up. If a task completes successfully without any problems, it should stay silent. That is the norm in Gradle.

Reply:
I guess I just want to always run with INFO then.